sopel/modules/translate.py

```diff
@@ -241,12 +241,13 @@ def get_random_lang(long_list, short_list):
 def mangle(bot, trigger):
     """Repeatedly translate the input until it makes absolutely no sense."""
     long_lang_list = ['fr', 'de', 'es', 'it', 'no', 'he', 'la', 'ja', 'cy', 'ar', 'yi', 'zh', 'nl', 'ru', 'fi', 'hi', 'af', 'jw', 'mr', 'ceb', 'cs', 'ga', 'sv', 'eo', 'el', 'ms', 'lv']
-    lang_list = []
+    lang_list: list[str] = []
```

Comment:
It works fine in all supported versions because [this module has deferred evaluation of annotations enabled](https://github.com/sopel-irc/sopel/pull/2471/files#diff-c4cfb1d787d3a60c348c827395dad95c16ab5b63a529efd15628591c42c88c63R9) (and the type checker knows about that), but what you say is true if we didn't have that future enabled.

Reply:
Modified to use `typing` in 28218c6